Makefile.work

```diff
@@ -217,15 +230,21 @@ SLAVE_TAG = $(shell \
 	(cat $(SLAVE_DIR)/Dockerfile.user \
 		 $(SLAVE_DIR)/Dockerfile \
 		 $(SLAVE_DIR)/buildinfo/versions/versions-* \
-		 .git/HEAD \
+		.git/HEAD 2>/dev/null \
```

Comment:
>.git/HEAD [](http://example.com/codeflow?start=2&length=9)

PR build will not have chance to use cache, can we remove it?

/_work/27/s$ cat .git/HEAD 
ref: refs/heads/master-12001

In current PR build, it will use the <branch-name>-<PR id> as the name.

Reply:
done